doc/script_commands.txt

```diff
@@ -3197,6 +3197,70 @@ within the specified area - an x1/y1-x2/y2 square on the specified map.
 This is useful for maps that are split into many buildings, such as all the
 "*_in" maps, due to all the shops and houses.
 
+---------------------------------------
+
+*getunits(<type>{,<array_variable>})
+*getmapunits(<type>,<"map name">{,<array_variable>})
+*getareaunits(<type>,<"map name">,<x1>,<y1>,<x2>,<y2>{,<array_variable>})
+
+'getunits' function will return the number of objects of the type passed currently whole server.
```

Comment:
`The 'getunits' command will return the number of <type> objects active on the server.`